qiskit_experiments/visualization/plotters/curve_plotter.py

```diff
@@ -101,7 +101,7 @@ def _default_figure_options(cls) -> Options:
 
         Figure Options:
             report_red_chi2_label (str): The label for the reduced-chi squared entry of the fit
-                report. Defaults to "reduced-$\\chi^2$`.
+                report. Defaults to "reduced-$\\\\chi^2$`.
```

Comment:
This is still rendering incorrectly:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/3870315/211235828-f63d51ce-2e3e-48af-95b7-330e0e95836d.png">

Reply:
I've addressed the docstring issue in dffbfd1. It now looks like this.
![Screenshot 2023-01-09 at 13 39 09](https://user-images.githubusercontent.com/6696704/211311497-b5a79c63-1f20-46ac-82b2-6a9db71a1cae.png)
The formatting of the `report_red_chi2_label` value in the Sphinx dropdown menu is “incorrect” in that it shows the processed value of the option and not the Python literal. This is owing to the way option defaults are processed by Qiskit Experiments Sphinx extensions, and would be an issue for other options and classes in the library.

Reply:
Perhaps it can be clarified with something like, Defaults to the expression rendered from "..."? I don't want users to think it literally looks like that.

Reply:
I understand. I think it's important for users to be aware of what the Python literal is. I've updated the description so that both the literal and the formatted string are shown. What do you think?
![Screenshot 2023-01-10 at 17 28 21](https://user-images.githubusercontent.com/6696704/211607770-53a847e7-f2aa-4f57-880f-0d25f7da2cc8.png)

Reply:
That's even better, thanks!